community/contribute/adding-docs-guide.md

```diff
@@ -0,0 +1,72 @@
+# Adding Docs Guide
+
+When you have landed here, most probably because you have documentation, which should be part of the SCS Documentation.
+
+In this Guide you will learn how to integrate your documentation to the SCS documentation, which you will find on docs.scs.community.
```

Comment:
Maybe shorter sentences:
In this Guide you will learn how to integrate your documentation to the SCS documentation.
The SCS documentation you can find on docs.scs.community

The docs.scs.community part as a link back to the frontpage of the documentation will give a better navigation.